pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -586,4 +586,10 @@ self: super: builtins.intersectAttrs super {
   # Tests require internet
   dhall_1_24_0 = dontCheck super.dhall_1_24_0;
 
+  jsaddle-wkwebview = overrideCabal super.jsaddle-wkwebview (old: {
+    broken = false;
```

Comment:
Remove `jsaddle-wkwebview` from the list of broken packages in `configuration-hackage2nix.nix` instead.